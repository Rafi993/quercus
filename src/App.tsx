import React from 'react';

const App: React.FC = () => {
  return <h1>Querqus</h1>;
};

export default App;
