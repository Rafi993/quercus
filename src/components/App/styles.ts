import styled from 'styled-components';

export const App = styled.div`
  > div:last-child {
    display: grid;
    grid-template-columns: 8rem 1fr;
  }
`;
