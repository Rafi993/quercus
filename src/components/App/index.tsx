import React from 'react';

import GlobalStyle from '../../styles';
import SideBar from '../SideBar';
import { App as StyledApp } from './styles';

const App: React.FC = () => {
  return (
    <>
      <GlobalStyle />
      <StyledApp>
        <SideBar />
      </StyledApp>
    </>
  );
};

export default App;
