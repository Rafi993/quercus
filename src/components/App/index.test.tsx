import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import App from './index';

afterEach(cleanup);

jest.mock('../SideBar', () => {
  const MockSideBar = () => <div />;
  return MockSideBar;
});

it('Testing if it contains sidebar component is rendered', () => {
  const { asFragment } = render(<App />);
  expect(asFragment()).toMatchSnapshot();
});
