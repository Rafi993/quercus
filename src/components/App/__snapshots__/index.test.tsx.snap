// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing if it contains sidebar component is rendered 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa gPPEKz"
  >
    <div />
  </div>
</DocumentFragment>
`;
